developer/src/kmc-model-info/src/model-info-compiler.ts

```diff
@@ -105,9 +105,9 @@ export function writeMergedModelMetadataFile(
   setModelMetadata('name', options.kmpJsonData.info.name.description);
 
   let author = options.kmpJsonData.info.author;
-  setModelMetadata('authorName', author.description);
+  setModelMetadata('authorName', author?.description);
 
-  if (author.url) {
+  if (author?.url) {
```

Comment:
Fixed a potential problem observed in packages in the keyboards repo although it hasn't yet arisen in the lexical-models repo.